### ISSUE TITLE
Records ordering is broken in avo >= 3.9.1

### ISSUE THREAD

**Issue by HusseinMorsy** [avo-hq/avo#2877]
### Describe the bug

Records ordering is broken avo >=  3.9.1. If you press lower/higher/top/bottom results in a 

**No route matches [PATCH] "/avo-pro/reorder/....** 

Exception. 

### Steps to Reproduce
1. Create a new rails application with avo-pro
2. Add "acts_as_list" gem
3. Add a model with `acts_as_list`
4. Add order configuration to the book resource
5. On the avo indes of the resource press the higher/lower buttton
6. See error

### Expected behavior & Actual behavior

Action on lower/upper/top/down should   route to `/avo/avo-pro/reorder/...` instead of just `/avo-pro/reorder/...`

### Models and resource files

Example Model:
bin/rails g model book title position:integer
Add "acts_as_list" to the book model

```ruby
class Book < ApplicationRecord
  acts_as_list
end
```

Add ordering to the avo book resource:

```ruby
class Avo::Resources::Book < Avo::BaseResource
  self.ordering = {
    visible_on: :index,
    actions: {
      higher: -> { record.move_higher },
      lower: -> { record.move_lower },
      to_top: -> { record.move_to_top },
      to_bottom: -> { record.move_to_bottom },
    }
  }
  self.includes = []
  # self.search = {
  #   query: -> { query.ransack(id_eq: params[:q], m: "or").result(distinct: false) }
  # }

  def fields
    field :id, as: :id
    field :title, as: :text
    field :author, as: :text
    field :position, as: :number
  end
end

```

### System configuration
**Avo version**: 3.9.2

**Rails version**: 7.1.3.3

**Ruby version**:3.3.2

**License type**:

 - [ ] Community
 - [X] Pro
 - [ ] Advanced

**Are you using Avo monkey patches, overriding views or view components?**

 - [ ] Yes. If so, please post code samples.
 - [X] No

### Screenshots or screen recordings

 - [ ] High impact (It makes my app un-usable.)
 - [x] Medium impact (I'm annoyed, but I'll live.)
 - [ ] Low impact (It's really a tiny thing that I could live with.)

#### Urgency

 - [ ] High urgency (I can't continue development without it.)
 - [X] Medium urgency (I found a workaround, but I'd love to have it fixed.)
 - [ ] Low urgency (It can wait. I just wanted you to know about it.)

Only workaround is to use avo 3.8.2

**Comment by adrianthedev** [comment]
Hey @HusseinMorsy,

You must have updated from an older version, or maybe updated Rails too.
Rails had a bug in 7.1 that we helped fix, but this means that Avo won't work with 7.1.

[This guide](https://docs.avohq.io/3.0/upgrade.html#upgrade-from-3-7-4-to-3-9-1) explains what you need to update and you'll find a link to the explanation and bug report as well.

TL;DR;

Update to Rails 7.2.0.beta2 or Rails `main`, and use custom `ransack` and `acts-as-taggable-on` gems like in the guide.

You're issue will be fixed.

**Comment by HusseinMorsy** [comment]
Thank you very much for your help and very fast response
